integration-tests/basic-test-app/app/_layout.tsx

```diff
@@ -1,25 +1,31 @@
 import {DarkTheme, DefaultTheme, ThemeProvider} from "@react-navigation/native";
 import {useFonts} from "expo-font";
-import {Stack} from "expo-router";
+import {Stack, useNavigationContainerRef} from "expo-router";
 import * as SplashScreen from "expo-splash-screen";
 import {useEffect, useMemo, useState} from "react";
 import "react-native-reanimated";
 import {initialize as initEmbrace} from "@embrace-io/react-native";
 
 import {useColorScheme} from "@/hooks/useColorScheme";
+import {useEmbraceNativeTracerProvider} from "@embrace-io/react-native-tracer-provider";
+import {NavigationTracker} from "@opentelemetry/instrumentation-react-native-navigation";
 
 // Prevent the splash screen from auto-hiding before asset loading is complete.
 SplashScreen.preventAutoHideAsync();
 
 export default function RootLayout() {
   const [embraceSDKLoaded, setEmbraceSDKLoaded] = useState<boolean>(false);
+  const {tracerProvider} = useEmbraceNativeTracerProvider({}, embraceSDKLoaded);
+  const navigationRef = useNavigationContainerRef();
+
   useEffect(() => {
     const init = async () => {
       const hasStarted = await initEmbrace({
         sdkConfig: {
           ios: {
-            appId: "abcdf",
+            appId: "cvKeD",
```

Comment:
do we want to commit the real app id of your test app?

Reply:
reverting for now, we may want to revisit this as part of integration testing as to what we want this to default to